frontend/app/filters/index.js

```diff
@@ -32,6 +32,8 @@ export function durationHumanize(duration) {
 export function scheduleHumanize(schedule) {
   if (schedule === null) {
     return 'Never';
+  } else if (schedule.split(' ').length === 5) {
+    return `CRON tab: ${schedule}`;
```

Comment:
How about returning "Advanced Schedule" and having a pop over show description of the schedule?